src/components/Header/Header.jsx

```diff
@@ -2,36 +2,68 @@ import React, { useState } from "react";
 import { Link } from 'react-router-dom';
 import classNames from "classnames";
 import icon_close from "../../assets/icon-close.svg";
+import icon_hamburger from "../../assets/icon-hamburger.svg";
 import logo from "../../assets/logo.svg";
 import styles from "./Header.module.css";
 
 const Header = () => {
-  const [mobileMenu, setMobileMenu] = useState(false);
+  const [mobileMenuOpen, setMobileMenuOpen] = useState(false);
   const toggleMenu = () => {
-    setMobileMenu((toggleMenu) => !toggleMenu);
+    setMobileMenuOpen(!mobileMenuOpen);
   };
 
-  const listClass = classNames(styles["nav-list"], {
-    [styles["hide-mobile-menu"]]: mobileMenu,
-  });
+  const standardNavClass = classNames(styles.headerNav,
+    styles.standardHeaderNav,
+  );
+
+  const mobileNavClass = classNames(styles.headerNav,
+    styles.mobileHeaderNav,
+    { [styles.hideMobileMenu]: !mobileMenuOpen, }
+  );
+
+  const showOrHideHamburger = classNames(styles.icon_hamburger, {
+    [styles.showHamburger]: !mobileMenuOpen,
+  })
+
+  const navLinks = (
+    <>
```

Comment:
This empty-looking `<>` tag opens a React Fragment, and is useful for returning one containing element to React without adding extra `div` tags to the HTML. The closing tag is `</>` and can be seen on line 39.